R/add_p.R

```diff
@@ -193,7 +193,7 @@ add_p <- function(x, test = NULL, pvalue_fun = NULL,
       ),
```

Comment:
Can you also updated `result$test <- "McNemar\\'s test"` in the examples section?